src/reconcile/rec_col.c

```diff
@@ -323,6 +339,8 @@ __wt_rec_col_int(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REF *pageref)
         if (page_del != NULL)
             WT_TIME_AGGREGATE_MERGE(session, &r->cur_ptr->ta, &ft_ta);
         WT_TIME_AGGREGATE_MERGE(session, &r->cur_ptr->ta, &ta);
+        if (__wt_process.page_stats_2022)
+            WT_PAGE_STAT_UPDATE(&r->cur_ptr->ps, &ps);
```

Comment:
Sometimes we use merge, sometimes we use update. They are really just two functions with mostly similar functionality with a little difference. Can we combine them?

Reply:
Yes, I think you're correct we can probably merge the two macros into one.